install_config/configuring_authentication.adoc

```diff
@@ -65,9 +65,64 @@ Basic authentication challenges are only sent if a `X-CSRF-Token` header is
 present on the request. Clients that expect to receive Basic `WWW-Authenticate`
 challenges should set this header to a non-empty value.
 
-.^|`login`     | When *true*, unauthenticated token requests from web clients
+|`login`     | When *true*, unauthenticated token requests from web clients
 (like the web console) are redirected to a login page backed by this provider.
 Not supported by all identity providers.
+
+|`mappingMethod`  | Defines how new identities are mapped to users when they login. See link:#mappingmethod-parameter[below] for more information.
+|===
+
+[NOTE]
+If you change authentication identity providers, but want to keep the same set
```

Comment:
Now that we have `mappingMethod`, they no longer need to keep the same provider name. If they want identities from the new provider to map to users that already exist, they need to set `mappingMethod` to `add`.